src/Backend/Modules/Blog/DataFixtures/LoadBlogPosts.php

```diff
@@ -2,39 +2,44 @@
 
 namespace Backend\Modules\Blog\DataFixtures;
 
+use Backend\Core\Language\Locale;
+use Backend\Modules\Blog\Domain\Category\Category;
+use Backend\Modules\Blog\Domain\Comment\Comment;
+use Common\Doctrine\Entity\Meta;
 use SpoonDatabase;
+use Symfony\Bundle\FrameworkBundle\Client;
 
 class LoadBlogPosts
 {
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
     public function load(SpoonDatabase $database): void
     {
-        $metaId = $database->insert(
-            'meta',
-            [
-                'keywords' => 'Blogpost for functional tests',
-                'description' => 'Blogpost for functional tests',
-                'title' => 'Blogpost for functional tests',
-                'url' => 'blogpost-for-functional-tests',
-            ]
+        $meta = new Meta(
+            'Blogpost for functional tests', false,
```

Comment:
all parameters need their own line

Reply:
I think PSR-2 needs this all on separate lines, style ci won't like this I think

Reply:
fixed in 4c322d4485af9cf7eb3496e8b351045e61c6ff60